bindata/etcd/cluster-restore.sh

```diff
@@ -75,11 +75,11 @@ if [ ! -d "$MANIFEST_STOPPED_DIR" ]; then
 fi
 
 # Move static pod manifests out of MANIFEST_DIR
-find ${MANIFEST_DIR} \
-  -maxdepth 1 \
-  -type f \
-  -printf '...stopping %P\n' \
-  -exec mv {} ${MANIFEST_STOPPED_DIR} \;
+for POD_FILE_NAME in "${STATIC_POD_LIST[@]}" etcd-pod.yaml; do
```

Comment:
Do we need to rethink the general logic here? This feels like we are bolting the solution on a bit I want to make sure we are not glazing over anything else that should be addressed in this PR.

Reply:
I can hold this PR, if you have a different solution. 

We definitely can't move everything out of the manifests directory and only replace 4 pods. So what we currently have definitely needs a fix.